database/PatientCheckInDBScript.sql

```diff
@@ -0,0 +1,143 @@
+create database PatientCheckIn
+--
+go
+--
+use PatientCheckIn
+--
+go
+--
+create table Patient (
```

Comment:
the script must handle even when the table exists so other people/automated runner can run the multiple times w/o issue